Android SDKManager doesn't run with JDK 17

<!--- Provide a brief summary of the issue -->
Substrate runs:
`$GRAALVM_HOME/bin/java ... com.android.sdklib.tool.sdkmanager.SdkManagerCli ...`
With JDK 17, it produces this exception:

```
$JAVA_HOME/bin/java -Dcom.android.sdklib.toolsdir=/home/jpereda/.gluon/substrate/Android/tools -classpath /home/jpereda/.gluon/substrate/Android/tools/lib/*:/home/jpereda/.gluon/substrate/Android/tools/lib/java11/* com.android.sdklib.tool.sdkmanager.SdkManagerCli platforms;android-30
Feb 03, 2022 12:21:32 AM com.sun.xml.bind.v2.runtime.reflect.opt.Injector <clinit>
SEVERE: null
java.security.PrivilegedActionException: java.lang.NoSuchMethodException: sun.misc.Unsafe.defineClass(java.lang.String,[B,int,int,java.lang.ClassLoader,java.security.ProtectionDomain)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:573)
	at com.sun.xml.bind.v2.runtime.reflect.opt.Injector.<clinit>(Injector.java:197)
...
Caused by: java.lang.NoSuchMethodException: sun.misc.Unsafe.defineClass(java.lang.String,[B,int,int,java.lang.ClassLoader,java.security.ProtectionDomain)
	at java.base/java.lang.Class.getMethod(Class.java:2227)
	at com.sun.xml.bind.v2.runtime.reflect.opt.Injector$3.run(Injector.java:201)
	at com.sun.xml.bind.v2.runtime.reflect.opt.Injector$3.run(Injector.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	... 46 more
```

To solve the issue, either update the dependencies, or try the new Command-Line Tools:

https://dl.google.com/android/repository/commandlinetools-linux-7302050_latest.zip

that doesn't need extra dependencies.

--- a/src/main/java/com/gluonhq/substrate/util/FileDeps.java
+++ b/src/main/java/com/gluonhq/substrate/util/FileDeps.java
@@ -71,12 +71,12 @@ public final class FileDeps {
 
     private static final String[] ANDROID_DEPS = {
             "https://repo1.maven.org/maven2/javax/activation/activation/1.1.1/activation-1.1.1.jar",
-            "https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-xjc/2.3.2/jaxb-xjc-2.3.2.jar",
-            "https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb-impl/2.3.0.1/jaxb-impl-2.3.0.1.jar",
-            "https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-core/2.3.0.1/jaxb-core-2.3.0.1.jar",
-            "https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-jxc/2.3.2/jaxb-jxc-2.3.2.jar",
+            "https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-xjc/2.3.3/jaxb-xjc-2.3.3.jar",
+            "https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb-impl/2.3.6/jaxb-impl-2.3.6.jar",
+            "https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-core/2.3.6/jaxb-core-2.3.6.jar",
+            "https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-jxc/2.3.3/jaxb-jxc-2.3.3.jar",
             "https://repo1.maven.org/maven2/javax/xml/bind/jaxb-api/2.3.1/jaxb-api-2.3.1.jar",
-            "https://repo1.maven.org/maven2/com/sun/istack/istack-commons-runtime/3.0.10/istack-commons-runtime-3.0.10.jar" };
+            "https://repo1.maven.org/maven2/com/sun/istack/istack-commons-runtime/3.0.11/istack-commons-runtime-3.0.11.jar" };
 
     private static final String ANDROID_KEY = "24333f8a63b6825ea9c5514f83c2829b004d1fee";
     private static final String[] ANDROID_SDK_PACKAGES = {
@@ -456,6 +456,7 @@ public final class FileDeps {
         String[] cliArgs = new String[] {
                 Paths.get(configuration.getGraalPath().toString(), "bin", "java").toString(),
                 "-Dcom.android.sdklib.toolsdir=" + tools,
+                "--illegal-access=permit",
                 "-classpath", libs + "/*:" + additionalLibs + "/*",
                 "com.android.sdklib.tool.sdkmanager.SdkManagerCli"
         };

